examples/with-sign-in-up-split-emailpassword/src/App.js

```diff
@@ -29,7 +29,7 @@ SuperTokens.init({
     recipeList: [
         EmailPassword.init({
             signInAndUpFeature: {
-                disableDefaultImplementation: true, // this disables showing the default sign in + sign up component
+                disableDefaultUI: true, // this disables showing the default sign in + sign up component
```

Comment:
Non tests example apps should not have this change since they get the auth-react SDK from npm and the code in their folder still refers to an older version of the SDK which has  `disableDefaultImplementation`